config/locales/en.txt

```diff
@@ -1629,6 +1637,7 @@
   rss_log_title: Activity Log
   rss_by: by [user]
   rss_title: "[:app_title]"
+  rss_filtered_mouseover: "[:app_title] has applied filter(s), including any filter(s) in: (a) your Preferences and (b) Advanced Search or the search bar."
```

Comment:
Sorry to be a pest about this, but the "(a) ... and (b) ... or ..." is hard for me to parse.  How do filters get applied through the search bar?  I must have missed that.  Would it work to just say "[:app_title] has applied filter(s), including any filter(s) in your preferences or search parameters."?